Lib/test/test_ctypes/test_wintypes.py

```diff
@@ -38,6 +41,19 @@ def test_variant_bool(self):
         vb.value = []
         self.assertIs(vb.value, False)
 
+    def assertIsSigned(self, ctype):
+        self.assertLess(ctype(-1).value, 0)
+
+    def test_signedness(self):
+        for type in (wintypes.BYTE, wintypes.WORD, wintypes.DWORD,
```

Comment:
Please rename type to ctype to avoid conflict with the built-in method.  While not an error here, many IDEs and linters will flag it as such.